### ISSUE TITLE
[RFC] Support inference with TVM

### ISSUE THREAD

**Issue by zhiqwang** [zhiqwang/yolov5-rt-stack#28]
## 🚀 Feature

Now, torchvision faster-rcnn and mask-rcnn can be [deployed](https://tvm.apache.org/docs/tutorials/frontend/deploy_object_detection_pytorch.html#sphx-glr-tutorials-frontend-deploy-object-detection-pytorch-py) with Relay VM, as we can understand that faster-rcnn and `yolov5rt` share the same operators. It could be deployed with minor changes.

## Additional context

https://discuss.tvm.apache.org/t/error-from-compiling-and-running-retinanet-from-torchvision/8678

**Comment by zhiqwang** [comment]
With recently updates of `tvm` and refactor of backbone (maybe this is not necessary), the model can be compiled and run successfully now. Wrapping a model as following:

```python
class TraceWrapper(nn.Module):
    def __init__(self, model):
        super().__init__()
        self.model = model

    def forward(self, inp):
        out = self.model(inp)
        return dict_to_tuple(out[0])

model_func = torch.hub.load('zhiqwang/yolov5-rt-stack', 'yolov5s', pretrained=True)
model = TraceWrapper(model_func)
```

I'll add more detailed information soon.

**Comment by Kyrie-Zhao** [comment]
aten::silu not implemented in relay frontend

**Comment by zhiqwang** [comment]
> aten::silu not implemented in relay frontend

Hi @Kyrie-Zhao , Please upgrade tvm to version 0.8 or above. Or you can load yolort model as belows:

```python
from yolort.models import yolov5s
model = yolov5s(export_friendly=True)
model = model.eval()
```
